arrabbiata/src/witness.rs

```diff
@@ -142,8 +142,11 @@ pub struct Env<
     /// The current iteration of the IVC
     pub current_iteration: u64,
 
-    /// A previous hash, encoded in 2 chunks of 128 bits.
-    pub previous_hash: [u128; 2],
+    /// The digest of the last program state, including the cross-terms
+    /// commitments.
+    /// The value is a 128 bits value, to be absorbed to initialize the sponge
```

Comment:
Not sure yet about this. I will see later. It is mostly to initialize the sponge state when absorbing the commitments to the state in the next PR.